test/WorkflowCore.TestAssets/LockProvider/DistributedLockProviderTests.cs

```diff
@@ -19,10 +20,10 @@ public void Setup()
             Subject.Start();
         }
 
-        protected abstract IDistributedLockProvider CreateProvider();        
+        protected abstract IDistributedLockProvider CreateProvider();
 
         [Test]
-        public async void AcquiresLock()
+        public async Task AcquiresLock()
```

Comment:
When I was running the tests, NUnit was complaining that these tests were invalid because they were `async void` instead of `async Task`. When you have `async void`, it does not give NUnit the hooks it needs to wait for the test to complete.